### PR TITLE
Feat: added Home Page

### PR DESCRIPTION
**Pull request by Nishant-96**
Added Home Page.

home1

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  margin-top: 6rem;
+}
+
+
+::-webkit-scrollbar {
+  display: none;
+}
+
+.landing-img-container{
+  margin-top: 5.6875rem;
```
Comment (muskaanshah) [inline index.css:108]: Why such a specific value? Try keeping max 1 decimal point

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    <main  class="landing-img-container">
+    
+            <div class="background-img-wrapper">
+                <img class="image-responsive rr-image-responsive" src="/Images-Ecom/rr-landing-background.jpg" alt='Landing Background Image'/>
+            </div>
+            <div class="landing-title-wrapper">
+                <h1>Welcome to Robyn Reads</h1>
+                <h3>Explore the World of Comics Here.</h3>
+                <button class="btn btn-primary landing-btn" onclick="window.location.href='/Screens/Product-Listing/productlisting.html';" >Explore Now</button>
```
Comment (muskaanshah) [inline index.html:36]: Try avoiding inline JS

File: index.css [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    height: auto;
+    z-index: 3;
+}
+
+.header-logo{
+    width:5.625rem;
```
Comment (muskaanshah) [inline index.css:45]: Why such a specific value? Try keeping max 1 decimal point

**Review by znjs — APPROVED** [review]

**Review by muskaanshah — APPROVED** [review]

File: index.css (hunk 1 above)
Comment (Nishant-96) [reply index.css:108]: its in order to place container just below nav without any space. Will fix it. Thank you.

File: index.css (hunk 3 above)
Comment (Nishant-96) [reply index.css:45]: Will fix it. Thank you.

File: index.html (hunk 2 above)
Comment (Nishant-96) [reply index.html:36]: That is done temporarily. I had planned to remove that while implementing functionality. Thank you for pointing it out.

**Pull request merged**